src/lib.rs

```diff
@@ -65,6 +65,12 @@ lazy_static! {
 ///                `<img width="" height="" src="" alt="">`
```

Comment:
This list is now somewhat incomplete. Should we try to keep it in sync or just link to the setters?